centos2ol.sh

```diff
@@ -381,7 +381,7 @@ for reponame in ${enabled_repos}; do
         elif [ "${action[0]}" == "RPM" ] ; then
             matching_rpm=${action[1]}
             echo "Installing ${matching_rpm} to get content that replaces ${reponame}"
-            yum install -y "${matching_rpm}"  --disablerepo "*" --enablerepo "ol*_latest"
+            yum install --assumeyes "${matching_rpm}"  --disablerepo "*" --enablerepo "ol*_latest"
```

Comment:
Again into the super nitpick territory: the syntax for `yum` commands is `yum [global options] <command> [command parameters]` so this command should be:

`yum --assumeyes --disablerepo "*" --enablerepo "ol*_latest" install "${matching_rpm}"`